src/goto-programs/abstract-interpretation/CMakeLists.txt

```diff
@@ -1,9 +1,9 @@
-add_library(abstract-interpretation ai.cpp ai_domain.cpp interval_domain.cpp interval_analysis.cpp)
+add_library(abstract-interpretation ai.cpp ai_domain.cpp interval_domain.cpp interval_analysis.cpp common_subexpression_elimination.cpp)
```

Comment:
I would prefer shorter file names. There could be a comment at the top of the .h or .cpp file saying "cse stands for [...]". The reason is that these long paths show up in the debugger everywhere and it gets harder to set breakpoints or look at stack traces.

Reply:
Thanks, Rafeal.